src/lib/rule-finder.js

```diff
@@ -71,21 +72,21 @@ function RuleFinder(specifiedFile) {
 
   // get all the current rules instead of referring the extended files or documentation
   this.getCurrentRules = function getCurrentRules() {
-    return currentRules
+    return sortRules(currentRules)
```

Comment:
I would have named `sortRules` to `getSortedRules` as its is not sorting in-place but returning the sorted list.